src/werkzeug/datastructures.py

```diff
@@ -1844,6 +1844,19 @@ def _normalize(language):
 
         return item == "*" or _normalize(value) == _normalize(item)
 
+    def best_match(self, matches, default=None):
+        """Calls the super version of `best_match` and if it returns none,
+        attempts to fall back to language only matches.
+
+        :param matches: a list of matches to check for
+        :param default: the value that is returned if none match
+        """
+        result = super(LanguageAccept, self).best_match(matches, default=default)
```

Comment:
Needs to be `default=None` here so the fallback works.

Test cases for `TestLanguageAccept`, below:
```
accept = self.storage_class([("en-us", 1)], default="en-us")
assert accept.best_match(["en"]) == "en"

accept = self.storage_class([("en-us", 1)], default="en-us")
assert accept.best_match(["de"]) == "en-us"
```

Reply:
I added your tests and simply took out passing the default to the super method.